modules/coreos-layering-configuring-on.adoc

```diff
@@ -57,32 +96,31 @@ spec:
     name: <mcp_name> <1>
   buildInputs:
     containerFile: # <2>
-    - containerfileArch: noarch <3>
+    - containerfileArch: noarch
       content: |-
-        FROM configs AS final <4>
-        RUN dnf install -y cowsay && \
-         dnf clean all && \
-         ostree container commit
-    imageBuilder: # <5>
+        FROM configs AS final
+        RUN rpm-ostree install tree && \
+            ostree container commit
+    imageBuilder: # <3>
       imageBuilderType: PodImageBuilder
-    baseImagePullSecret: # <6>
+    baseImagePullSecret: # <4>
       name: global-pull-secret-copy
-    renderedImagePushspec: image-registry.openshift-image-registry.svc:5000/openshift/os-image:latest  # <7>
-    renderedImagePushSecret: # <8>
+    renderedImagePushspec: image-registry.openshift-image-registry.svc:5000/openshift/os-image:latest  # <5>
+    renderedImagePushSecret: # <6>
       name: builder-dockercfg-7lzwl
-  buildOutputs: # <9>
+  buildOutputs: # <7>
```

Comment:
This is not in the `v1` API. It is from the outdated `v1alpha` API

Reply:
According to [David's comment above](https://github.com/openshift/openshift-docs/pull/87730#issuecomment-2663957330)  and my testing, 4.18 uses vialpha1